supporting_content/attributes/column_naming_convention.md

```diff
@@ -16,3 +16,57 @@ Examples of column names found in provider datasets:
 - Group wanted to get naming convention adopted broadly for provider columns as well, to simplify reporting and analysis by practitioners
 - Different conventions like camel case, snake case, and title case were considered
 - Discussion included different columns naming conventions used by databases
+
+### How to handle custom columns
```

Comment:
I'm trying to get a sense of these rules.  Would any of the following be allowed: 

- "aws_linelitem/LineItemDescription" (mixed us of "_" and "/")
- "aws_lineitem_LineItemDescription" (not strictly pascal case)
- "aws_LineItemDescription" (does not match precise name in AWS data dictionary)

Reply:
All we're saying in this PR is that every vendor MUST prefix their columns with some recognizable lowercase + underscore prefix. We already say they SHOULD use the same rules as we do, but that also means they do any of the 3 options you mentioned. The goal of "SHOULD" was to make provider adoption simpler. In general, we've thought about starting with "SHOULD" and moving some of those to "MUST" in the second release.

Reply:
UPDATE: Based on feedback in Slack (and since nobody seems to want to sign off on the vendor prefix), I removed the vendor prefix. Now the requirement is to add a single underscore (`_`). Everything else is the same. Resolving this, but feel free to reopen or leave another comment.